app/Models/Notification/Notification.php

```diff
@@ -5,20 +5,27 @@
 use App\Models\Controller\ControllerPosition;
 use App\Models\User\User;
 use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\BelongsToMany;
 use Illuminate\Database\Eloquent\SoftDeletes;
 
 class Notification extends Model
 {
+    use HasFactory;
```

Comment:
```suggestion
    use HasFactory, SoftDeletes;
```

Reply:
Iirc PSR12 doesn’t like multiple traits per use! 


https://www.php-fig.org/psr/psr-12/#42-using-traits

Reply:
The PHP docs aren't PSR12 compliant then :D

Reply:
They never are :D